.github/workflows/ci.yml

```diff
@@ -57,6 +57,40 @@ jobs:
       env:
         BUNDLE_GEMFILE: fixtures/Gemfile-github-pages
 
+  validate:
+    name: Validate HTML
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [3.1]
```

Comment:
We should add Ruby 3.2 at some point (I'm testing with `ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-linux]`)

Reply:
Sure; I'll update all of those (+ the template) in lockstep.